runtime/parachains/src/inclusion/mod.rs

```diff
@@ -524,11 +524,20 @@ impl<T: Config> Pallet<T> {
 				candidates.iter().enumerate()
 			{
 				if let FullCheck::Yes = full_check {
-					check_ctx.verify_backed_candidate(
+					match check_ctx.verify_backed_candidate(
 						parent_hash,
+						parent_storage_root,
 						candidate_idx,
 						backed_candidate,
-					)?;
+					)? {
+						Err(FailedToCreatePVD) => {
+							// We don't want to error out here because it will
+							// brick the relay-chain. So we return early without
+							// doing anything.
+							return Ok(ProcessedCandidates::default())
```

Comment:
If we want to play it safe - that is fine, but we should at least have logs. Otherwise it will be a nightmare to debug any issues.

Reply:
Sure. Note that this is the exact same logic as status quo and there are no logs. but I am fine with adding some. @Lldenaurois please add those and get this into ship-shape